<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="idNum">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="stdName">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="infoDisplay">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>171</y>
     <width>831</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Course ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Course Discrption</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Course Section #</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Capacity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Instructor ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Instructor Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Credits</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Semester</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Flags</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
